packages/tailwind-config/tailwind.config.ts

```diff
@@ -1,12 +1,13 @@
-/** @type {import('tailwindcss').Config} */
-module.exports = {
+import type {Config} from 'tailwindcss';
+
+const config = {
```

Comment:
Would it be possible to add `export default` to this line, so as to make immediately clear that it's being exported as a default?

Reply:
Only if we change this eslint rule: https://github.com/import-js/eslint-plugin-import/blob/v2.28.1/docs/rules/no-anonymous-default-export.md